app/controllers/api/vcf_files_controller.rb

```diff
@@ -89,7 +92,16 @@ def create
     service.job_id = job.id
     service.save
     respond_to do |format|
-      msg = { msg: MSG_VCF_SUCCESS_PEDIA_RUNNING }
+      if passed
+        msg = {msg: MSG_VCF_PASSED_QC }
+      else
+        if output == nil
```

Comment:
Use nil? instead of ==
and also use elsif instead of nested if
https://stackoverflow.com/questions/48731014/ruby-use-the-return-of-the-conditional-for-variable-assignment-and-comparison